projects/core/src/lib/components/toolbar/simple-search/simple-search.component.ts

```diff
@@ -89,14 +86,23 @@ export class SimpleSearchComponent implements OnInit, OnDestroy {
       pointType: 'circle',
       pointStrokeWidth: 0,
     });
-    this.mapService.renderFeatures$('search-result-highlight', of({
+    const feature: FeatureModel = ({
       __fid: 'search-result-highlight-feature',
       geometry: searchResult.geometry,
       crs: searchResult.projectionCode,
       attributes: {},
-    }), style, true, true).pipe(
-      takeUntil(timer(5000)),
-    ).subscribe();
+    });
+
+    this.mapService.getProjectionCode$().pipe(take(1))
```

Comment:
This is not how to use RxJS, you should not use a variable outside your stream. There should be a single stream to get the projectioncode and then use switchMap/concatMap to render the features.